include/grpcpp/security/credentials_impl.h

```diff
@@ -146,6 +154,9 @@ struct SslCredentialsOptions {
   /// This parameter can be empty if the client does not have a certificate
   /// chain.
   std::string pem_cert_chain;
+
+  /// The TLS version negotiated during the TLS handshake.
```

Comment:
I'm not sure the comment is quite correct. How about the following: `The range of TLS versions allowed by the client.`